app/api_topology_test.go

```diff
@@ -67,13 +67,13 @@ func TestAPITopologyApplications(t *testing.T) {
 	defer ts.Close()
 	is404(t, ts, "/api/topology/applications/foobar")
 	{
-		body := getRawJSON(t, ts, "/api/topology/applications")
+		body := getRawJSON(t, ts, "/api/topology/containers")
```

Comment:
Why is TestAPITopologyApplications making a GET on /api/topology/containers? Can you move this to TestAPITopologyContainers?